src/Test.UnitTests.Sarif.Driver/PathExtensionsTests.cs

```diff
@@ -31,6 +31,17 @@ public PathExtensionsTests(ITestOutputHelper outputHelper)
             outputHelper.WriteLine($"TestName: {nameof(PathExtensionsTests)} has random seed {randomSeed}");
         }
 
+        [Fact]
+        [Trait(TestTraits.WindowsOnly, "true")]
+        public void ContainsInvalidPathChar_test()
+        {
+            string validPath = "test.md";
+            validPath.ContainsInvalidPathChar().Should().BeFalse();
+
+            string invalidPath = "test\t.md";
```

Comment:
>\t [](http://example.com/codeflow?start=38&length=2)

consider get an invald char from `System.IO.Path.GetInvalidPathChars()` so it should work on both windows/linux/mac